core/transformation/pathfinders/consts.py

```diff
@@ -8,27 +8,11 @@
 }
 
 PF_REPORTING_PERIOD_TO_DATES = {
```

Comment:
can you add an inline comment to detail exactly what this mapping is for (ie for translating dates present in the PF spreadsheet, I presume)